lib/polymer_interop.dart

```diff
@@ -1,102 +1,303 @@
+/// to update its distribution.
+///
+/// Other resources: https://www.polymer-project.org/1.0/docs/devguide/local-dom.html
+class PolymerDom {
+  final JsObject _proxy;
 
-  static Future import(List urls) {
-    if (!checkExists()) return null;
-    var completer = new Completer();
-    _polymer.callMethod('import', [urls, () => completer.complete()]);
-    return completer.future;
-  }
+  final Node node;
 
-  static void whenPolymerReady(f()) {
-    if (!checkExists()) return null;
-    _polymer.callMethod(
-        'whenPolymerReady', [Zone.current.bindCallback(() => f())]);
-  }
+  /// The insert, append, and remove operations are transacted lazily in certain
+  /// cases for performance. In order to interrogate the dom (e.g. offsetHeight,
+  /// getComputedStyle, etc.) immediately after one of these operations, call
+  /// PolymerDom.flush() first.
+  static void flush() => _Polymer['dom'].callMethod('flush');
+
+  PolymerDom(node)
+      : _proxy = _Polymer.callMethod('dom', [node]),
+        this.node = node;
+
+  /// Calling appendChild where the parent is a custom Polymer element adds the
+  /// node to the light DOM of the element.
+  ///
+  /// In order to append into the shadow root of a custom element, use
+  /// `this.root` as the parent.
+  ///
+  /// The appendChild method is transacted lazily in certain cases for
+  /// performance. In order to interrogate the dom (e.g. offsetHeight,
+  /// getComputedStyle, etc.) immediately after one of these operations, call
+  /// PolymerDom.flush() first.
+  Node append(node) => _proxy.callMethod('appendChild', [node]);
+
+  /// Read-only property that returns a live list of child nodes of the given element.
+  ///
+  /// The list also includes e.g. text nodes and comments. To skip them, use
+  /// `this.children` instead.
+  List<Node> get childNodes => _proxy['childNodes'];
+
+  /// Read-only property that returns a live `List` of the child elements of
+  /// `this`. The list doesn't include text nodes and comments.
+  List<Element> get children => _proxy['children'];
+
+  // TODO: add docs
+  PolymerClassList get classList => new PolymerClassList(this);
+
+  /// Read-only property that returns the node's first child in the tree, or
+  /// null if the node is childless.
+  Node get firstChild => _proxy['firstChild'];
+
+  /// Read-only property that returns the object's first child Element, or null
+  /// if there are no child elements.
+  Element get firstElementChild => _proxy['firstElementChild'];
 
-  static void endOfMicrotask(f()) {
-    if (!checkExists()) return null;
-    _polymer.callMethod('endOfMicrotask', [() => f()]);
+  /// Sometimes it’s necessary to access the elements which have been distributed
+  /// to a given <content> insertion point or to know to which <content> a given
+  /// node has been distributed. The getDistributedNodes and
+  /// getDestinationInsertionPoints methods, respectively, provide this information.
+  List<Node> getDestinationInsertionPoints() =>
```

Comment:
https://www.polymer-project.org/1.0/docs/devguide/local-dom.html talks a bit more about these, but it doesn't specify the behavior of getDistributedNodes if you have more than one `<content>` element.